bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/matcher/EncounterSessionMatcherTest.java

```diff
@@ -146,9 +148,9 @@ public void setUp(){
 
         when(userContext.getAuthenticatedUser()).thenReturn(creator);
 
-        when(encounterService.getEncounters(any(Patient.class), any(Location.class),
+        when(encounterService.getEncounters(any(Patient.class), eq(null),
```

Comment:
here again the expectations seem to be different. Before it was some location object, now thats null .. any reason?

Reply:
the value that is being passed test run time is "null". If we don't assert with "null" the test case is failing. With earlier mockito it was not checking test run time value.